### PR TITLE
Initial commit, added folder structure

### PR DESCRIPTION
**Pull request by palani-ND**
Added basic folder structure
- src
  - config
    - config.js
  - controllers
  - middleware
  - models
  - routes
  - services
  - utils
  - index.js
- .gitignore
- package.json

### CODE REVIEW

File: .DS_Store
Comment (Hari-1027) [inline .DS_Store:1]: this config is not needed. this is Mac's cache file . add them to ignore

File: package-lock.json
Comment (Hari-1027) [inline package-lock.json:1]: package-lock should never added to git.

File: src/index.js
Comment (Hari-1027) [inline src/index.js:1]: why is this index here?

Comment (Hari-1027) [inline src/index.js:1]: where are we adding the services, controllers, models?

File: package.json
Comment (Hari-1027) [inline package.json:1]: a PR at the development should always be from a fork. no Branch to branch PR is encouraged.

Comment (Hari-1027) [reply package.json:1]: here is the doc on this point Bachi shared - https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow

Comment (palani-ND) [reply package.json:1]: okay @Hari-1027 will do that 👍🏽

File: .DS_Store
Comment (palani-ND) [reply .DS_Store:1]: okay

File: package-lock.json
Comment (palani-ND) [reply package-lock.json:1]: okay

**Pull request closed without merging**